src/shared-styles/_validation.scss

```diff
@@ -0,0 +1,33 @@
+$is-test: false !default;
+
+@function error($message, $capture: $is-test) {
+  @if $capture {
+    @return $message;
+  }
+
+  @error $message;
+}
+
+@mixin error($message) {
+  _error: error($message);
+}
```

Comment:
Maybe these should be their own module?

Reply:
Tried that but I couldn't work out how to pass in `$is-test` with `with`, as `error` is then `@use`d by `validation` which needs to provide a different `$is-test` value depending on the calling context. Might just be a sass modules learning point, but going to leave it for now until a clear pattern emerges.

Reply:
**UPDATE**: Got it working in https://github.com/libero/storybook/pull/35.